tik/lib/libtik.cpp

```diff
@@ -1,10 +1,9 @@
 #include "tik/libtik.h"
-#include "tik/Kernel.h"
 #include <map>
 #include <memory>
 namespace TraceAtlas::tik
 {
     llvm::Module *TikModule;
-    std::map<int64_t, std::shared_ptr<Kernel>> KernelMap;
-    std::map<llvm::Function *, std::shared_ptr<Kernel>> KfMap;
+    std::map<int64_t, std::shared_ptr<CartographerKernel>> KernelMap;
+    std::map<llvm::Function *, std::shared_ptr<CartographerKernel>> KfMap;
```

Comment:
Why did you replace this?

Reply:
But are those attributes used? You shouldn't need to access those components if they weren't before? Or am I missing something?